### ISSUE TITLE
Tetrad Keeps considering data as continuous variable when validating data

### ISSUE THREAD

**Issue by khaisieu1995** [cmu-phil/tetrad#1192]
Tetrad Keeps considering date data as continuous variable when validating data even when you select "mixed data" with high limits

**Comment by kvb2univpitt** [comment]
Is it possible to provide a sample of the dataset?

**Comment by khaisieu1995** [comment]
[Bike-Sharing-Dataset.zip](https://github.com/cmu-phil/tetrad/files/3497550/Bike-Sharing-Dataset.zip)

This is the publicly available data set.  If you simply take a subset that contains the dates, temp, hum, atemp, and count. Tetrad will not be able to tell that dates can be discrete. You would need to force all discrete or all continuous (which will give an error).

Nevertheless, I have found a solution: you just have to load the dates alone to Tetrad and validate it as a discrete variable. Then, copy and paste the continuous columns in by hand.

**Comment by kvb2univpitt** [comment]
Thanks for the link to the dataset.  I am glad you've found a work-around.  We'll take a look at it and try to fix it in the meanwhile.

**Comment by kvb2univpitt** [comment]
@khaisieu1995 Since the dates are all unique, setting "max discrete categories" high would not work.  The right way would be to provide a metadata file as shown below:

```javascript
{
  "domains": [
    {
      "name": "dteday",
      "discrete": true
    }
  ]
}
```

However, even that doesn't work with the data validation.  Need to fix the data validation.

**Comment by kvb2univpitt** [comment]
Fixed.

**Comment by khaisieu1995** [comment]
Dear Kevin, would you mind sending the link to the download the version of tetrad where this issue is fixed?
I dont believe it is fixed in 6.7.0.
Thank you very much!

**Comment by kvb2univpitt** [comment]
@khaisieu1995 The changes have been made to the development branch.
